docs/docs/non-cloud-deployment.md

```diff
@@ -84,32 +76,9 @@ A quick summary:
     * For Ubuntu 20 LTS: `echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' > /etc/apt/sources.list.d/intelsgx.list`
     * Add the Intel package signing key: `wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -`
     * Run `apt-get update`
-    * Run `apt-get install libssl-dev libcurl4-openssl-dev libprotobuf-dev libsgx-urts libsgx-launch libsgx-epid libsgx-quote-ex`
+    * Run `apt-get install libssl-dev libcurl4-openssl-dev libprotobuf-dev libsgx-urts libsgx-launch libsgx-quote-ex`
```

Comment:
The integratlon-tests-build container has epid reference

Reply:
You mean in the `Dockerfile`, right?
Sorry, my IntelliJ does not pick up all the files when switching branches, removing that install package as well as well.